openapi.yaml

```diff
@@ -145,6 +145,13 @@ components:
             $ref: '#/components/schemas/RouteStep'
         annotation:
           $ref: '#/components/schemas/Annotation'
+        via_waypoints:
+          type: array
+          nullable: true
+          default: null
+          description: A valhalla and mapbox extension for via waypoints.
```

Comment:
Can we add a bit more detail? I couldn't understand what this does from the current description. Based on https://github.com/valhalla/valhalla/blob/master/src/tyr/serializers.cc#L371, it sounds like it's a set of geometry indexes that correspond to any intermediate waypoints between the origin and destination.